tests/test_filters.py

```diff
@@ -2375,6 +2375,21 @@ def test_extrude_rotate():
         and (ymax == line.bounds[1])
     )
 
+    rotation_axis = (0, 1, 0)
+    f = pyvista._vtk.vtkRotationalExtrusionFilter()
+    if not hasattr(f, 'SetRotationAxis'):
```

Comment:
If you switch to using vtk version you would need to change here too.

Reply:
```suggestion
    if pyvista.vtk_version_info < (9, 1, 0):
```